packit_service/models.py

```diff
@@ -176,6 +176,33 @@ def get_projects(cls, first: int, last: int) -> Iterable["GitProjectModel"]:
             )[first:last]
             return projects
 
+    @classmethod
+    def get_namespace(cls, forge: str, namespace: str) -> Iterable["GitProjectModel"]:
+        """Return projects of given forge and namespace"""
+        with get_sa_session() as session:
+            projects = (
+                session.query(GitProjectModel).filter_by(namespace=namespace).all()
+            )
+            matched_projects = []
+            for project in projects:
+                forge_domain = urlparse(project.project_url).hostname
+                if forge == forge_domain:
+                    matched_projects.append(project)
+            return matched_projects
+
+    @classmethod
+    def get_project(
+        cls, forge: str, namespace: str, repo_name: str
+    ) -> Optional["GitProjectModel"]:
+        """Return one project which matches said criteria"""
+        with get_sa_session() as session:
+            project = cls.__choose_project(
+                session=session, forge=forge, namespace=namespace, repo_name=repo_name
+            )
+            if not project:
+                return None
```

Comment:
`__choose_project` already returns `None`. Could this just `return project`—without the conditional?